morpheus/_lib/cudf_helpers.pyx

```diff
@@ -69,7 +72,20 @@ cdef public api:
                 name = c_name.decode()
                 column_names.append(name if name != "" else None)
 
-        data, index = data_from_table_view(table_info.table_view, owner=owner, column_names=column_names, index_names=index_names)
+
+        column_indicies = []
+
+        for c_index in table_info.column_indices:
+            print("c_index:", c_index)
+            column_indicies.append(c_index)
+
+        x = table_info.table_view
+
+        try:
+            data, index = data_from_table_view_mine(x, owner=owner, column_names=column_names, column_indices=column_indicies, index_names=index_names)
+        except Exception:
+            import traceback
+            print(traceback.format_exc())
```

Comment:
Improve the print statement with a prefix like "Error converting table when calling..."